ddtrace/ext/priority.py

```diff
@@ -13,12 +13,13 @@
 # Indicate to keep the trace
 span.context.sampling_priority = USER_KEEP
 """
+from ddtrace.constants import AUTO_KEEP
+from ddtrace.constants import AUTO_REJECT
+from ddtrace.constants import USER_KEEP
+from ddtrace.constants import USER_REJECT
 
-# Use this to explicitly inform the backend that a trace should be rejected and not stored.
-USER_REJECT = -1
-# Used by the builtin sampler to inform the backend that a trace should be rejected and not stored.
-AUTO_REJECT = 0
-# Used by the builtin sampler to inform the backend that a trace should be kept and stored.
-AUTO_KEEP = 1
-# Use this to explicitly inform the backend that a trace should be kept and stored.
-USER_KEEP = 2
+
+USER_REJECT = USER_REJECT
+AUTO_REJECT = AUTO_REJECT
+AUTO_KEEP = AUTO_KEEP
+USER_KEEP = USER_KEEP
```

Comment:
You don't need to remap these, they are available as-is from the imports.

What you are probably missing is:

```python
__all__ = ["USER_REJECT", "AUTO_REJECT", "AUTO_KEEP", "USER_KEEP"]
```

To tell flake8/other tools that those are imported and meant to be unused since they will be re-exported from this module.

Reply:
Ahh didn't know about that! Will change to use `__all__ = ["USER_REJECT", "AUTO_REJECT", "AUTO_KEEP", "USER_KEEP"]`.